webpack/assets/javascripts/react_app/routes/common/WithRenderHandler/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import EmptyPage from '../EmptyPage';
+import LoadingPage from '../LoadingPage';
+
+const WithRenderHandler = (
+  Component,
+  LoadingComponent = LoadingPage,
+  EmptyComponent = EmptyPage
+) => componentProps => {
```

Comment:
It would be nice to use:
```js
const WithRenderHandler = ({
  Component,
  LoadingComponent = LoadingPage,
  EmptyComponent = EmptyPage
})
```

So I can only supply what I want to override `WithRenderHandler({ Component, EmptyComponent  })`